runtime/static_config_test.go

```diff
@@ -400,7 +432,7 @@ func TestCanReadFromFileContents(t *testing.T) {
 	expected := testStruct{
 		Boolean: true,
 		Integer: 1,
-		Float:   1.0,
+		Float:   1.2,
```

Comment:
There was some custom code added to `MustGetInt` I think @l-w-2017 intended to add a test case for `MustGetInt` that would fail for `1.2` but pass for `1.0`.

Reply:
Jacob is right. "1.0" can be converted to an integer. "SetConfigValueOrDie" support a type "number" which is stored as a float.